packages/venia-concept/src/components/Icon/icon.js

```diff
@@ -13,8 +13,8 @@ class Icon extends Component {
     };
 
     render() {
-        const { classes, name } = this.props;
-        const svg = Feather.icons[name].toSvg();
+        const { attrs, classes, name } = this.props;
+        const svg = Feather.icons[name].toSvg(attrs);
```

Comment:
Note: This `feather-icons` library is [64.67kb minified](https://unpkg.com/feather-icons@4.7.3/dist/), and forces every icon into our bundle, along with a bunch of duplicated Babel helpers. Gzipped the lib is still 15.7kb. We should look for an alternative

Reply:
For what it's worth, never used an icon set on client project. At most they use 10 icons and still need another 10 arrows/misc figures we have to hand-draw. It's simpler to draw/pluck them without importing a massive icon set.